WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -36,10 +39,12 @@ final class OrderDetailsViewModel {
 
     let productRightTitle = NSLocalizedString("QTY", comment: "Quantity abbreviation for section title")
 
-    /// Anything above 999.99 or below -999.99 should display a truncated amount
+    /// The localized unabbreviated total which includes the currency.
+    ///
```

Comment:
Is it an empty comment line wanted?

Reply:
Ah, yes. The first empty line is to separate the [Summary sentence (first line) from the Discussion paragraph](https://nshipster.com/swift-documentation/) (the `"Example..."`). 

The last empty line is to follow the convention in the code. It looks like we prefer to add an empty line before starting the `func` declaration.